scripts/deploy.js

```diff
@@ -0,0 +1,97 @@
+#! /usr/bin/env node
```

Comment:
This file could use formatting corrections and a bunch of comments to explain what's going on. We can do it together if you like.

Reply:
done.